keps/sig-network/3668-reserved-service-nodeport-range/README.md

```diff
@@ -467,5 +467,6 @@ N/A
 
 - v1.27 - Initial implementation https://github.com/kubernetes/kubernetes/pull/114418
 - v1.28 - Beta
+- v1.29 - GA
```

Comment:
update the GA criteria please

> #### GA
> - No issues reported during two releases.

to `no issues reported during one release`. , we can clarify that this is because this feature is built on top of a previous feature, so we don´t. need the additional soak time

Reply:
Thanks, It's updated.

Reply:
Thanks, It's updated.